docs/getting-started.md

```diff
@@ -106,6 +106,18 @@ The result file will contain information about the `mime-types` package itself,
 information about each dependency. The scope names come from the package managers, for NPM packages these are usually
 `dependencies` and `devDependencies`, for Maven package it would be `compile`, `runtime`, `test`, and so on.
 
+By default, `analyzer-result.yml` will not contain any information, about scopes that can be excluded in later steps (for example, to skip downloading and scanning `devDependencies`). To specify, which scopes should be skipped, add `.ort.yml` configuration file to the root directory of the repo. For more details on config files see [Configuration File](config-file-ort-yml.md).
```

Comment:
Can we maybe phrase this in a more "positive" way, like "Note that the `analyzer-result.yml` is supposed to capture all known information about a project, which can then be "filtered" in later steps. For example, scopes which are not relevant for the distribution will still be listed, but can be configured to get excluded so that they e.g. do not get downloaded and scanned by the scanner step. To specify which scopes should be excluded, add an `.ort.yml` configuration file to the input directory of the analyzer. For more details see [Configuration File](config-file-ort-yml.md)."

Reply:
This sounds better, especially clarification of `analyzer-result.yml is supposed to capture all known information about a project, which can then be "filtered" in later steps.` Updated my PR.